guides/v2.1/cloud/project/project-multi-sites.md

```diff
@@ -64,128 +59,145 @@ To configure routes in an integration environment:
 		"https://*.{default}/":
     		type: upstream
     		upstream: "mymagento:php"
-5.	Save your changes to `routes.yaml` and exit the text editor.
-
-### Set up websites, stores, and store views {#cloud-multi-stores-admin}
 
-Set up in your {{site.data.var.ee}} Admin websites, stores, and store views identical to the ones you set up on your local system.
+1.  Save your changes to the `routes.yaml` file.
 
-#### Get your access information
+### Set up websites, stores, and store views
 
-To get the access information you need to log in to the Magento Admin:
+Set up your {{site.data.var.ee}} Admin websites, stores, and store views identical to the ones you set up on your local workstation.
```

Comment:
This seems a little misleading... you set theses things up _using_ the Admin panel.